app/domain/authentication/authn_jwt/signing_key/fetch_signing_key_interface.rb

```diff
@@ -2,7 +2,7 @@ module Authentication
   module AuthnJwt
     module SigningKey
       class FetchSigningKeyInterface
-        def call; end
+        def fetch_signing_key; end
```

Comment:
add the function `signing_key_uri`

Reply:
Interfaces will be removed from authn jwt because they are not relevant in ruby